.flake8

```diff
@@ -1,6 +1,7 @@
 [flake8]
 # Based directly on Black's recommendations:
 # https://black.readthedocs.io/en/stable/the_black_code_style.html#line-length
-max-line-length = 80
+max-line-length = 800
```

Comment:
Seems like a long line! What's this based on?

Reply:
That's based on committing debug lines like an idiot. (Although in my defence `git diff master` was massive)

Reply:
Fixed in the other PR